stack/rush-stack/gulpfile.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 const build = require('@microsoft/node-library-build');
-
+build.mocha.enabled = false;
```

Comment:
Intentional?

Reply:
Yep, these projects don't have tests and the new mocha throws if you don't have any tests..